lisp/forge-post.el

```diff
@@ -56,6 +56,15 @@
                        (forge-issue   'issue-post-url-format)
                        (forge-pullreq 'pullreq-post-url-format))))
 
+(cl-defmethod forge-browse :after
+  ;; always mark the post as read
+  ((post forge-post))
```

Comment:
The comment seems misplaced *and* unnecessary.
Also please place the arguments on the same line as `:after`.

Reply:
I'll take care of the comment – you're right, it probably isn't necessary. (There was a point where the implementation was split within the same file, so it seemed more helpful there. No longer.)

Placing the arguments on the same line messes up indentation pretty badly – review other uses of `:after` and friends in Emacs core.

Reply:
Well I'll eat my hat – apparently the `:after` thing has been fixed (either by updating something or by restarting emacs). I've updated indentation accordingly.

Reply:
I vaguely remember indentation being wrong at some point too. I worked around it by doing it manually and was a bit surprised when that wasn't necessary anymore.